src/templates/en/2019/chapter.html

```diff
@@ -1,5 +1,7 @@
-{% extends "en/base.html" %}
+{{% extends "en/2019/base_chapter.html" %}}
 
-{% block title %}
-Chapter {{ chapter }} | The Web Almanac by HTTP Archive
-{% endblock %}
+{{% set metadata = {{ {metadata} | json }} %}}
```

Comment:
Sorry, the JSON string replacement removes the need for `| json`.

```suggestion
{% set metadata = {metadata} %}
```